pyanaconda/payload/dnf/payload.py

```diff
@@ -1992,10 +1992,19 @@ def _import_rpm_keys(self):
             # TODO: replace the interpolation with DNF once possible
             arch = util.execWithCapture("uname", ["-i"]).strip().replace("'", "")
             vers = util.get_os_version(conf.target.system_root)
+
+            if not os.path.exists(conf.target.system_root + "/usr/bin/rpm"):
+                log.error("Can not import GPG keys to RPM database because rpm is missing. The "
```

Comment:
```suggestion
                log.error("Can not import GPG keys to RPM database because rpm tool is missing. The "
```
It's a bit unclear otherwise.

Reply:
Bikeshedding of course, but an [RPM tool](https://www.rpmtool.com/) is something else :grin: (I've never seen rpm to be referred to as a "tool".)

Reply:
hmm... good point. When someone tell me RPM, I always first think about some package. I guess that's the problem on my side?

Reply:
Oh! Now I understand how you read it before :) I. e. "some random rpm is missing", not "rpm the package manager" is missing -- i.e. `rpm.rpm`. Interesting semantics. So, just ignore me then, it's fine for me either way.

Reply:
Thanks, good catch! I think you want "executable".